### DISCUSSION TITLE
Aria2 compared to rTorrent

### DISCUSSION THREAD

**Discussion by mirolm** [jesec/flood#398] [Ideas]
I use rtorrent with flood and it works really awesome. I'm just wandering how is aria2 client compared to rtorrent?

I know aria2 is very lightweight and have xmlrpc/json interface. Anyone tried it and how it is compared to rtorrent and other clients?

If it is better how hard it will be to see aria2 support in flood?

**Comment by KaKi87** [comment]
@jesec Could we have your opinion on this ? Thanks

**Comment by luckydevil13** [comment]
+1 for aria2

**Comment by trim21** [comment]
aria2 is a good BT client but not pt client

**Comment by tribhuwan-kumar** [comment]
+1

**Comment by jesec** [comment]
aria2 supports downloads other than bittorrent (i.e. HTTP, FTP, etc.).

At this moment, Flood in general does not support other types of downloads well. We may add the support but that requires a revamp on the API, data types, etc.

**Comment by kumarvivek1752** [comment]
+1
